plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreConfig.java

```diff
@@ -39,6 +39,7 @@
     private Duration maxBackoffDelay = RetryDriver.DEFAULT_SLEEP_TIME;
     private Duration maxRetryTime = RetryDriver.DEFAULT_MAX_RETRY_TIME;
     private boolean impersonationEnabled;
+    private boolean sparkStatisticsFallbackEnabled = true;
```

Comment:
Do we want to add a corresponding session property ?

Reply:
If thrift metastore cache is enabled, this would not work well and would add a lot of complexity to this.